draft-chen-cfrg-vdaf-pine.md

```diff
@@ -185,7 +185,7 @@ The user-specified parameters to initialize PINE are defined in
 |:----------------|:------|:---------------|
 | `l2_norm_bound` | float | The L2-norm upper bound (inclusive). |
 | `dimension`     | int   | Dimension of each gradient. |
-| `num_frac_bits` | int   | The number of bits of precision to use when encoding each gradient entry into the field. At most XXX |
+| `num_frac_bits` | int   | The number of bits of precision to use when encoding each gradient entry into the field. At most 127. |
```

Comment:
The upper bound depends on the field and norm bound, correct?

Perhaps we should just say nothing here.
```suggestion
| `num_frac_bits` | int   | The number of bits of precision to use when encoding each gradient entry into the field. |
```